test/refactoring/rename/localEnum.swift

```diff
@@ -0,0 +1,14 @@
+func boop() {
+  enum LocalEnum {
+    case north
+    case south
+  }
+
+  print(LocalEnum.north)
```

Comment:
Some more tests:
  - Define enum in a struct (add another scope level)
  - Add another with the print line before the def rather than after (this shouldn't make a difference but...)